modules/nf-core/salsa2/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process SALSA2"
+    script "../main.nf"
+    process "SALSA2"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "salsa2"
+
+    test("Single-Read") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end: false ], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['genome']['genome_fasta_fai'], checkIfExists: true)
+                ]
+                input[1] = [
+                    file(params.test_data['sarscov2']['genome']['test_bed'], checkIfExists: true)
+                ]
+                input[2] = []
+                input[3] = []
+                input[4] = []
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert snapshot(process.out.fasta[0][1]).match("32c805e9e447ad47d6437e10f91bd5bf") },
+                { assert snapshot(process.out.agp[0][1]).match("03fe8c559b80730aab465697a6b0e01c") },
```

Comment:
```suggestion
                { assert snapshot(process.out.fasta[0][1]).match("fasta") },
                { assert snapshot(process.out.agp[0][1]).match("agp") },
```

better to use more meaningful match names as above. If you also agree plz regenerate the snapshots. You are almost there ;)